src/createServer.js

```diff
@@ -1,3 +1,33 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { validate } = require('./validate');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+    const toCase = normalizedURL.searchParams.get('toCase');
+    const pathname = normalizedURL.pathname.slice(1);
+    const errors = validate(pathname, toCase);
+
+    if (errors.length) {
+      req.statusCode = 400;
+      req.statusMessage = 'Bad request';
+      res.end(JSON.stringify({ errors }));
+    } else {
```

Comment:
Shouldnt we hande default "OK' case?

Reply:
👍

Reply:
Don't we already have `200 Ok` by default?

Reply:
Sorry, you are right it is default